static/js/components/ExpandedPostDisplay.js

```diff
@@ -153,17 +194,19 @@ export default class ExpandedPostDisplay extends React.Component<*, void> {
     return (
       <div className="post-summary expanded">
         <div className="summary">
-          <ProfileImage
-            profile={{
-              name:        post.author_name,
-              image_small: post.profile_image
-            }}
-            useSmall={true}
-          />
           <div className="post-title">{formatPostTitle(post)}</div>
           <div className="authored-by">
-            by <span className="author-name">{post.author_name}</span>,{" "}
-            {formattedDate}
+            <div className="left">
+              <ProfileImage
+                profile={{
+                  name:        post.author_name,
+                  image_small: post.profile_image
```

Comment:
Here, because `useMicro` is specified `makeProfileImageUrl` ends up returning the default profile image because the `useSmall` value passed to it is `undefined` and there is no medium profile image passed here.